www_src/pages/page/page.jsx

```diff
@@ -35,14 +31,29 @@ var Page = React.createClass({
   },
 
   componentWillMount: function() {
+    // who are we?
+    var params = this.state.params;
+    this.uri =  `/users/${params.user}/projects/${params.project}/pages/${params.page}`;
```

Comment:
this will break if you change users/change projects etc. without fully reloading the page